sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/ModelSettingsClient.cs

```diff
@@ -89,13 +87,12 @@ public virtual Response<TimeSeriesModelSettings> GetModelSettings(CancellationTo
         /// <param name="name">Model display name which is mutable by the user. Initial value is &quot;DefaultModel&quot;.</param>
         /// <param name="cancellationToken">The cancellation token.</param>
         /// <returns>The updated model settings with the http response <see cref="Response{TimeSeriesModelSettings}"/>.</returns>
-        public virtual async Task<Response<TimeSeriesModelSettings>> UpdateModelSettingsNameAsync(string name, CancellationToken cancellationToken = default)
+        public virtual async Task<Response<TimeSeriesModelSettings>> UpdateNameAsync(string name, CancellationToken cancellationToken = default)
         {
-            using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(TimeSeriesInsightsClient)}.{nameof(UpdateModelSettingsName)}");
+            using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(TimeSeriesInsightsClient)}.{nameof(UpdateName)}");
             scope.Start();
             try
             {
-                // To do: Generate client session Id
                 var options = new UpdateModelSettingsRequest { Name = name };
                 Response<ModelSettingsResponse> modelSettings = await _modelSettingsRestClient.UpdateAsync(options, null, cancellationToken).ConfigureAwait(false);
```

Comment:
need to pass in client session id in this call.

Reply:
removing the client session id all together per our offline conversation